From cbfee3fb2f03f83a9bedbe4af07048cae7a104fa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 23 Apr 2019 14:09:18 +0200
Subject: [PATCH 11/29] meson: do not use f-strings

Our travis CI still uses python3.5. I'm making this into a separate
commit to make it easy to revert later.

(cherry picked from commit 155dc16168fd98a8bfddcf7fd4ceba472d44db51)
---
 src/test/generate-sym-test.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/test/generate-sym-test.py b/src/test/generate-sym-test.py
index 2510809452..4d358b8e34 100755
--- a/src/test/generate-sym-test.py
+++ b/src/test/generate-sym-test.py
@@ -13,9 +13,9 @@ for line in open(sys.argv[1]):
     if match:
         s = match.group(1)
         if s == 'sd_bus_object_vtable_format':
-            print(f'    &{s},')
+            print('    &{},'.format(s))
         else:
-            print(f'    {s},')
+            print('    {},'.format(s))
 
 print('''};
 
-- 
2.21.0

